static/templates/test/or.hbs

```diff
@@ -0,0 +1,2 @@
+{{#if (or)}}<p>empty or</p>{{/if}}
+{{#if (or false last)}}<p>last or</p>{{/if}}
```

Comment:
We should have an `or true` case as well, just like and.

Reply:
Added a case for this.